templates/pages/category.html

```diff
@@ -25,9 +25,15 @@ <h1 class="page-heading">{{category.name}}</h1>
 {{{snippet 'categories'}}}
 <div class="page">
     {{#or category.subcategories category.faceted_search_enabled category.shop_by_price}}
```

Comment:
I think the if/elseif below can be moved into this #or block helper. We import most of https://github.com/helpers/handlebars-helpers#helpers so you can use #or and #and together to achieve the same thing. Try this:
`#or category.subcategories category.faceted_search_enabled (and category.shop_by_price theme_settings.shop_by_price_visibility)`

Reply:
Try this instead:
```
{{#any category.subcategories category.faceted_search_enabled (all category.shop_by_price theme_settings.shop_by_price_visibility)}}
```

Reply:
both have the "TypeError: Uncaught error: opts.inverse is not a function" error

Reply:
Ok, forget that, I guess you need an else clause for that to work. I would just go back to your current solution, but can you get rid of the redundant `#or` that wraps all this?